_posts/2015-09-18-epages-you-2015.md

```diff
@@ -0,0 +1,103 @@
+---
+layout: post
+title: "Annual company event: ePagees rock the YOU"
+date: "2015-09-18 07:21:00"
+icon: users
+categories: events
+authors: ["Birgit"]
+---
+
+Creative team presentations, team building activities, workshops and lot's of fun: last weeks' activities and corridor talks at ePages all revolved around the ePages annual company event: the YOU.
+The YOU is a formal, annual company event which all ePages employees are required to attend.
+This sounds really tough.
+But it is not.
+It is incredibly fun, although it is hard work, too and involves long hours.
+
+Main goal of the event:
+
+* get to know each other better
+* unite colleagues from all company locations
+* exchange ideas among colleagues, departments and countries.
+
+# Event task force
+
+Already a long time before the actual date of the YOU, an event task force started their activities to prepare the ground for two awesome days.
+To make it a big surprise for everyone, the agenda was kept a secret for a long time.
+But soon the time had come to position some teasers and involve the employees.
+
+# Prepare for action
+
+Every team was asked to prepare a 1-minute video, that presents their departments.
+To make it short: that was a hell of a lot of fun!
+We also had some restrictions for the team presentations itself: hold a 12-minutes presentation and prepare for a 12-minutes Q&A session.
+That was definitely a challenge and required time for a careful preparation.
+
+{% image blog-you-prepare.jpg 100% %}{% endimage %}
+
+# Patience required
+
+There were rumors that we would go for an outdoor activity.
+Soon you heard colleagues whisper and chatter on the corridors, what that might be.
+Climbing? Geocaching? City walk?
+No chance to find that out.
+Staying patient was required.
+
+# Kick-off at the beach
+
+{% image blog-you-beachclub.jpg 100% %}{% endimage %}
+
+Finally the YOU started in a beach club above the city roofs of Hamburg.
+We enjoyed a beautiful view across the city, along with food and drinks, a relaxed atmosphere and lots of fun.
+Incidentally, we also had the chance to observe a beautiful sundown!
+
+# Opening YO(U)
+
+The next day we all convened at the Hamburg University to spend the first day watching funny and awesome team videos and listening to well-prepared team presentations.
+
+{% image blog-you-presentation.jpg 100% %}{% endimage %}
+
+Exciting: which team was going to present was drawn right before the presentation, so every team had to be prepared at all times!
+
+# Dragon boat race
+
+In the afternoon the secret was finally lifted what would be our outdoor activity: we went to a small dock at a branch of the Alster to spend the afternoon with dragon boat races.
```

Comment:
In the afternoon, the secret was finally revealed as to what our outdoor activity would be. We were taken to a small dock at a branch of the Alster to spend the afternoon competing in dragon boat races.